integrationtests/tests/tests.rs

```diff
@@ -169,13 +169,10 @@ async fn ecash_cannot_double_spent_with_different_nodes() {
     let out3 = user3.client.reissue(ecash, rng()).await.unwrap();
     fed.run_consensus_epochs(2).await; // process transaction + sign new coins
 
-    assert!(
-        user2.client.fetch_coins(out2).await.is_err()
-            || user3.client.fetch_coins(out3).await.is_err()
```

Comment:
This was just a problem with `||` short-cutting and so `user3` wouldn't fetch coins.